src/bin/service_main.rs

```diff
@@ -143,146 +143,6 @@ async fn main() {
     println!("-> Service shut down.");
 }
 
-async fn get_file(
-    Path(project_name): Path<String>,
-    Path(file_name): Path<String>,
-    State(project_manager): State<Arc<Mutex<FlowProjectManager>>>,
-) -> Result<impl IntoResponse, (StatusCode, String)> {
-    let project = project_manager
-        .lock()
-        .unwrap()
-        .projects
-        .get(&project_name)
-        .cloned();
-    if let None = project {
-        let error_message = format!(
-            "The Project {} does not exist in the Project Manager",
-            project_name
-        );
-        eprintln!("{}", error_message);
-        return Err((StatusCode::INTERNAL_SERVER_ERROR, error_message.to_string()));
-    }
-    let project_path = PathBuf::from(format!("./flow-projects/{}", project_name));
-    let target_dir = project_path.join("pkg");
-
-    if !target_dir.exists() {
-        let error_message = "The target dir of the generated WASM file cannot be found.";
-        eprintln!("{}", error_message);
-        return Err((StatusCode::INTERNAL_SERVER_ERROR, error_message.to_string()));
-    }
-
-    let file_name = file_name;
-    let file_path = target_dir.join(file_name.clone());
-    let file = tokio::fs::File::open(file_path.clone())
-        .await
-        .expect("Failed to open Wasm file");
-    let stream = ReaderStream::new(file);
-    let body = StreamBody::new(stream);
-    let content_type = match mime_guess::from_path(file_path).first_raw() {
-        Some(mime) => mime,
-        None => {
-            return Err((
-                StatusCode::BAD_REQUEST,
-                "MIME Type couldn't be determined".to_string(),
-            ))
-        }
-    };
-    let headers = [
-        (header::CONTENT_TYPE, content_type.to_string()),
-        (
-            header::CONTENT_DISPOSITION,
-            format!("attachment; filename=\"{:?}\"", file_name),
-        ),
-    ];
-
-    Ok((headers, body))
-}
```

Comment:
@FWuermse 

Das funktioniert derzeit nicht. Welchen usecase hat diese Funktion ? 
Ich hätte sie mal gestrichen, da sie keinen usecase in der Website so hat, wenn Sie da nichts gegen hätten.